ui/apps/platform/src/Containers/Network/Header/NetworkSearch.js

```diff
@@ -64,6 +66,7 @@ function NetworkSearch({
     function onSearch(options) {
         setSearchFilter(options);
         if (isCompleteSearchFilter(options)) {
+            history.push(`/main/network${history.location.search}`);
             closeSidePanel();
```

Comment:
Out of curiosity, I commented out this line to see if it is still needed. The answer is yes.

For a while longer, two sources of truth are better than one ;)